docs/src/main/sphinx/connector/hive.rst

```diff
@@ -567,63 +567,66 @@ In order to use a Glue catalog, ensure to configure the metastore with
 ``hive.metastore=glue`` and provide further details with the following
 properties:
 
-==================================================== ============================================================
-Property Name                                        Description
-==================================================== ============================================================
-``hive.metastore.glue.region``                       AWS region of the Glue Catalog. This is required when not
-                                                     running in EC2, or when the catalog is in a different region.
-                                                     Example: ``us-east-1``
+======================================================= ============================================================
```

Comment:
nit: Updating table width separately from new content keeps diffs cleaner.

Reply:
Good point. Thank you for raising this aspect.